pkg/cli/environment.go

```diff
@@ -107,6 +104,10 @@ func New() *EnvSettings {
 		NoEmojis: false,
 	}
 
+	if err := env.checkCorrectHelmSettings(); err != nil {
+		log.Fatal(err)
+		os.Exit(1)
+	}
 	os.Setenv("HELM_NAMESPACE", env.namespace) // WHY???
```

Comment:
Out of context to this PR, but this is prevents helm code being run on helms data.